cdn.py

```diff
@@ -1,33 +1,65 @@
 """Turn local URLs into CDN URLs."""
-import os, sys
-import urllib
 import urlparse
-from nose.tools import set_trace
 
 from config import Configuration, CannotLoadConfiguration
-from s3 import S3Uploader
+from core.mirror import MirrorUploader
+from core.model import ExternalIntegration
+from core.s3 import S3Uploader
+
+
+def _get_mirrors():
```

Comment:
As we discussed earlier, you can make cdn.py forget about S3 entirely and remove this code.

Reply:
Removed the S3-related logic from `cdnfy`